app/views/layouts/application.html.erb

```diff
@@ -7,25 +7,42 @@
 
     <%= stylesheet_link_tag    'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
+
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV" crossorigin="anonymous"></script>
   </head>
 
   <body>
-    <nav class="navbar">
-      <a class="navbar-brand" href="#">GlitchCraft</a>
-      <div class="." id=".">
-        <ul class="navbar-page-links">
-          <li class="nav">
-            <a class="nav-link" href="/">Home</a>
-          </li>
-        </ul>
-        <ul class="navbar-login-links">
-          <li class="nav">
-            <a class="nav-link" href="/auth/google_oauth2">Login</a>
-          </li>
+    <nav class="navbar navbar-expand-lg navbar-dark" style="background-color: #000000;">
```

Comment:
![](https://media0.giphy.com/media/TBddd797slSxO/200.gif?cid=5a38a5a21oys9z2fbgojd20vkpa5wp9mlaa5hfnwmpju8rh6&rid=200.gif)
this is so fancy!